assets/js/download.js

```diff
@@ -15,7 +16,7 @@ $(document).ready(function() {
     else if (navigator.userAgent.match(/android/i)) {
       browser = 'other';
     }
-    else if (navigator.userAgent.match(/chrome\/.+edge?\//i)) {
+    else if ((m = navigator.userAgent.match(/chrome\/.+edge?\/(\d+)/i)) && parseInt(m[1]) >= 79) {
```

Comment:
Do we need to check version numbers? It looks like ChrEdge uses "Edg/" while legacy Edge uses "Edge/".